input/postgres/databases.go

```diff
@@ -8,60 +8,75 @@ import (
 	"github.com/pganalyze/collector/util"
 )
 
-const databasesSQLDefaultOptionalFields = "1"
-const databasesSQLpg93OptionalFields = "datminmxid"
+const databasesSQLDefaultMxidFields = "1"
+const databasesSQLpg93MxidFields = "d.datminmxid"
+const databasesSQLDefaultMxidAgeFields = "0"
+const databasesSQLpg93MxidAgeFields = "CASE WHEN d.datminmxid <> '0' THEN mxid_age(d.datminmxid) ELSE 0 END"
 
 // See also https://www.postgresql.org/docs/9.5/static/catalog-pg-database.html
 const databasesSQL string = `
-SELECT oid,
-			 datname,
-			 datdba,
-			 pg_catalog.pg_encoding_to_char(encoding),
-			 datcollate,
-			 datctype,
-			 datistemplate,
-			 datallowconn,
-			 datconnlimit,
-			 datfrozenxid,
-			 %s
-	FROM pg_catalog.pg_database`
+SELECT
+	d.oid,
+	d.datname,
+	d.datdba,
+	pg_catalog.pg_encoding_to_char(d.encoding),
+	d.datcollate,
+	d.datctype,
+	d.datistemplate,
+	d.datallowconn,
+	d.datconnlimit,
+	d.datfrozenxid,
+	%s,
+	CASE WHEN d.datfrozenxid <> '0' THEN age(d.datfrozenxid) ELSE 0 END,
+	%s,
+	(sd.xact_commit + sd.xact_rollback) AS transaction_count
```

Comment:
I think it'd be good if we keep these as separate counters in the collector and snapshot data - we may want to show them as separate graphs at some point, and it doesn't cost much in terms of snapshot size.

(we can still store them as one combined number in the application data model itself, but do that combination on the server side instead of collector side)

Reply:
We're not storing the transaction count itself to the pganalyze app, we're using this data to calculate TPS within the collector. Though, I've added `xact_commit` and `xact_rollback` separately to the message of database stats, and I'll simply not store these data in the pganalyze app side for now.